Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM progrium/busybox
+FROM alpine:latest
 MAINTAINER Stephane Jourdan <fasten@fastmail.fm>
-ENV REFRESHED_AT 2016-03-15
-ENV VAULT_VERSION 0.5.1
-ENV VAULT_TMP_FILE /tmp/vault.zip
+ENV REFRESHED_AT 2016-04-25
+ENV VAULT_VERSION 0.5.2
 
 # x509 expects certs to be in this file only.
-ADD https://raw.githubusercontent.com/bagder/ca-bundle/master/ca-bundle.crt /etc/ssl/certs/ca-certificates.crt
+RUN apk update && apk add openssl ca-certificates
 
-ADD https://releases.hashicorp.com/vault/${VAULT_VERSION}/vault_${VAULT_VERSION}_linux_amd64.zip ${VAULT_TMP_FILE}
```

Comment:
Adding a file is not a good option, since even though you delete the file later a layer is created and will be wasting image space anyway.